synapse/rest/client/v1/login.py

```diff
@@ -195,11 +196,11 @@ async def _do_other_login(self, login_submission):
             if medium is None or address is None:
                 raise SynapseError(400, "Invalid thirdparty identifier")
 
+            # For emails, canonicalise the address.
+            # We store all email addreses as cononised in the DB.
```

Comment:
"cononised" isn't a word.
```suggestion
            # We store all email addresses canonicalised in the DB.
```